httpie/ssl_.py

```diff
@@ -104,6 +104,18 @@ def __init__(
         self._verify = None
 
         if ssl_version or ciphers:
+            # By default, almost all installed CPython have modern OpenSSL backends
+            # This actively prevent folks to negotiate "almost" dead TLS protocols
+            # HTTPie wants to help users when they explicitly expect "old" TLS support
+            # Common errors for user if not set:
+            #   >- [SSL: NO_CIPHERS_AVAILABLE] no ciphers available
+            #   >- [SSL: LEGACY_SIGALG_DISALLOWED_OR_UNSUPPORTED] legacy sigalg disallowed or unsupported
+            if ssl_version in {ssl.PROTOCOL_TLSv1, ssl.PROTOCOL_TLSv1_1} and ciphers is None:
+                # Please do not raise a "security" concern for that line.
+                # If the interpreter reach that line, it means that the user willingly set
+                # an unsafe TLS protocol.
+                ciphers = "DEFAULT:@SECLEVEL=0"
```

Comment:
Q: just a quick check — the case of TLSv1(.1) with explicit --ciphers is not covered here; wouldn't you need the seclevel added for these as well, or you'd expect the user to add the at-rule themselves in that case?

Reply:
If the user put a cipher list himself, we should not interfere. HTTPie will interfere (user knows what he's doing)
adding SECLEVEL appended to his suites is going to bring confusion.

Reply:
short answer: yes we should let the user decide, or/and figure it out himself.

Reply:
Q: I've learned the ciphers are not DEFAULT but own constant set; yet here the seclevel string appends to a DEFAULT. is that what is expected?